rest/route/task_generate.go

```diff
@@ -47,6 +52,29 @@ func (h *generateHandler) Run(ctx context.Context) gimlet.Responder {
 	if err := data.GenerateTasks(h.taskID, h.files); err != nil {
 		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "generating tasks for task '%s'", h.taskID))
 	}
+	t, err := task.FindOneId(h.taskID)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "getting task '%s'", h.taskID))
+	}
+	if t == nil {
+		return gimlet.MakeJSONErrorResponder(errors.Errorf("task '%s' not found", h.taskID))
+	}
+	ts := utility.RoundPartOfMinute(1).Format(units.TSFormat)
+	j := units.NewGenerateTasksJob(t.Version, t.Id, ts)
+	queueName := fmt.Sprintf("service.generate.tasks.version.%s", t.Version)
+	queue, err := evergreen.GetEnvironment().RemoteQueueGroup().Get(ctx, queueName)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "getting generate tasks queue '%s' for version '%s'", queueName, t.Version))
+	}
+	grip.Info(message.Fields{
+		"message": "enqueueing generate tasks job",
+		"version": t.Version,
+		"task_id": t.Id,
+		"job_id":  j.ID(),
+	})
+	if err = amboy.EnqueueUniqueJob(ctx, queue, j); err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "enqueueing catchup job"))
```

Comment:
`"enqueueing generate tasks job"`

I think it's actually okay to just log a warning and continue, rather than return an error. This could fail for non-problematic reasons, like a race between this and the cron populator to populate the job (since after L52, the cron populator may already have detected the task needs generation). At this point, we've already persisted the generate tasks JSON state to the DB, so this is just a nice-to-have for speeding up the process.